### PR TITLE
Feature/forgot password

### PR DESCRIPTION
**Pull request by henriquepablo**
# Code Trouble - Pull Request (Mobile)

---

### **Description**

- **What was done?**  forgot password screen
- **Why was it done?**  to exists a forgot password page

---

### **Change Type**

- [ ] Bug fix  
- [x] New feature  
- [ ] Code improvement  
- [ ] Documentation  

---

#### **Platform Testing**
- [x] Have you tested on an Android emulator?  
- [ ] Have you tested on an iOS simulator?  
- [x] Have you tested on a physical Android device?  
- [ ] Have you tested on a physical iOS device?  

#### **Performance**
- [x] Have you verified app performance (e.g., animations, loading times)?  
- [x] Have you optimized assets (e.g., images, icons)?  

#### **Security**
- [x] Were tests added for new features or changes?  
- [ ] Was the documentation updated (if needed)?  

#### **Code Quality**
- [x] Have you followed the project’s linting and formatting rules?  
- [x] Are all commits clear and meaningful?  
- [ ] Have you ensured clean and modular code structure?  

#### **Accessibility**
- [x] Have you tested for screen reader compatibility?  
- [ ] Have you ensured proper contrast ratios and font scaling?  

---

### **Screenshots (if applicable)**

_Add screenshots of your changes on different devices/platforms if necessary._

---

{C9288DDD-A955-49DA-8791-E0B7056B99F8}

{428ED088-6D65-47C2-AC94-7C722CFD8EC6}

### CODE REVIEW

File: src/screens/auth/forgot-password/ForgotPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { AddBookmark, Hide, Logo, Show } from '@icons/index.ts'
```
Comment (lucascorreaa) [inline src/screens/auth/forgot-password/ForgotPassword.tsx:1]: Aulassss

File: src/screens/auth/forgot-password/style.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
+import { theme } from '@theme/theme.ts'
+import styled from 'styled-components/native'
+
+export const Container = styled.View<{ $large: boolean }>`
+    /* flex: 1; */
```
Comment (lucascorreaa) [inline src/screens/auth/forgot-password/style.ts:5]: Esse código comentado tem algum funcionamento?

**Review by lucascorreaa — APPROVED** [review]
Pra cima !

File: src/screens/auth/forgot-password/style.ts (hunk 2 above)
Comment (henriquepablo) [reply src/screens/auth/forgot-password/style.ts:5]: Não, nas próximas prs o senhor vai ver que refiz essas partes, então já foi retitado

**Pull request merged**